Wrappers/Python/cil/optimisation/functions/TotalVariation.py

```diff
@@ -313,6 +325,11 @@ def proximal(self, x, tau, out = None):
             if should_break:
                 break
         
+        # restore the values of inputs: tau , x
+        if self.strong_convexity_constant>0:
+            tau *= (1 + tau*self.strong_convexity_constant)
```

Comment:
Is your maths correct here? We have
```
t = tau/(1+ tau*g)
x = x / (1 + tau*g)
```

If at the end we want to recover the values of x that were initially passed by the user we need to recover what `tau` was from `t`, i.e.

```
tau = t / (1 - t*g)

```
with this `tau` the line below is correct.

Reply:
I think is better and more clear if we use a `tmp_tau`

Reply:
as long as it is a number

Reply:
Please resolve